### DISCUSSION TITLE
Issues on Provision Tenant

### DISCUSSION THREAD

**Discussion by drskur** [awslabs/aws-saas-boost#504] [Q&A]
I'm following the tutorial in the following link. (https://github.com/awslabs/aws-saas-boost/blob/main/docs/getting-started.md#onboard-a-tenant)

But CF fails with the following error
- Logical ID: InvokeRDSBootstrapDatabase
- reason
```
Received response status [FAILED] from custom resource. Message returned: software.amazon.awssdk.services.s3.model.S3Exception: Your key is too long (Service: S3, Status Code: 400, Request ID: JF7DFV70WV2X9VCR, Extended Request ID: cqFgYJumVMKZLUgJh7W/mWxKvvd6usETPYtOoLtTdY0dqNhpGU9EDKi6h7Lm1UyxyXEHa4KVsrM=) (RequestId: 83fdaaf1-e7b6-497a-823e-df5ef7dcbbff)
```

- Version: v2.2.0-11@0bd9b27

**Comment by mariandumitrascu** [comment]
This error happens usually when creating a file on S3 that is longer than accepted limit. The name of these files (the key) is constructed using some input data such as the name of the environment. Try to re-configure, giving shorter names for everything that is input. For example keep the environment name to a 3 or 4 letters long.
